### PR TITLE
Added frontend UI Component Testing

### PR DESCRIPTION
**Pull request by TZnoj**
Added UI Component tests for the front end.
This includes the fields and buttons for the login and registration page. These tests were done using Cypress.

### CODE REVIEW

File: frontend/cypress.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "baseUrl": "http://localhost:8081"
+    "baseUrl": "http://localhost:8080"
```
Comment (purplesmurf1998) [inline frontend/cypress.json:2]: This change may affect the capability of running the tests for some people if they have something running on that port already. Is there a fix for this?

Comment (AnthonyChraim) [reply frontend/cypress.json:2]: Yes, you are right. If you are not running on port 8080, cypress can not verify that the server is running. This gets fixed once docker gets set up

Comment (purplesmurf1998) [reply frontend/cypress.json:2]: Ok, we'll make sure to have docker set up in the next sprint so that we can fix issues like this.

**Review by purplesmurf1998 — APPROVED** [review]
Testing looks good. Using Cypress works in this case, but we should also look into using Vue.js component testing so that we can automate it.

**Pull request merged**